### ISSUE TITLE
Don't overwhelm searcher with all non-media records

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#410]
@mccalluc and @afred: Could we have two solr indices? One would be for all 2.5 million records, excluding those items that are digitized and online. The second solr index would be only digitized items that are online.

Users would use different facet lists and search results pages to search these two catalogs. 

On the homepage, a user would conduct a search and would be taken to the search results page for the Digitized and Online Solr index. On the homepage below the search box, there is a link to take the user to the solr index for all 2.5 million records. 

Karen and Alan would like to make the 2.5 million records less visible. They would prefer that these items be separate from the solr index for items that are Digitized and Online.

**Comment by afred** [comment]
What's the motivation behind making 2.5M records less visible? That seems to run counter to dissemination goals.

If the motivation is simply because we don't want search results to be too noisy, then I'd favor a solution that didn't involve splitting the index.

**Comment by caseyedavis12** [comment]
You're right, it's kind of contrary to what we're trying to do. We're trying to figure out a good user experience before we can actually fix the records for the non digitized assets.

The motivation is that the 2.5 million are extremely messy records. Primary user will be a user that doesn't care about an asset at a small station that is not digitized that they will never be able to use.

What do you think would be a good way to highlight as much as possible only items that users can view/listen to online, with second tier being all digitized assets, and third tier being records for non-digitized assets?

Splitting solr index was just an idea. I defer to you and Chuck on how best to technologically meet this goal.

**Comment by caseyedavis12** [comment]
Ok, we looked at Boston Local TV News Digital Library, and we think that we should use this same functionality for displaying digitized and non-digitized. (http://bostonlocaltv.org/catalog?utf8=%E2%9C%93&q=cooking&search_field=all_fields&button=)

Remove "access" filter from Facets list and put the checkboxes at the top of the search results page.

Digitized and Online -- Default 
Digitized and available on Location (checkbox)
Inventory Records for Non-Digitized Items (checkbox)

**Comment by afred** [comment]
that's funny.. @sroosa and I were just saying how we really don't like the way OV and Boston Local News do it :)

**Comment by caseyedavis12** [comment]
What alternative solution would you propose?

**Comment by afred** [comment]
What if we branded the "non digitized" assets as something other than just a facet value?

For instance, come up with a term that means something like "Backlog", or "Yet to be digitized", or "The deep dark recesses of the index".

Then we can take what would normally be a facet field, or a little checkbox, and give it the appropriate attention it deserves, from a UX standpoint.

My thought is that searching for A/V resources is a much, much different user experience than sifting through 2.5M text-only records. Given this big discrepancy, we shouldn't boil it down to just an incidental checkbox, or a facet field that looks like all the rest. It should be a clear button to switch into "deep dive" mode, a clear signal that you are currently in "deep dive" mode, and a clear button to get back to normal searching for A/V.

**Comment by afred** [comment]
this is the type of thing that @Hammicakes could probably give some good insight into.

**Comment by afred** [comment]
(and whatever we come up with for AAPB, we should consider porting it back to News and OV :smile:)

**Comment by caseyedavis12** [comment]
Agree. Sounds great. Will add this to the list for @hammicakes for future work.

**Comment by mccalluc** [comment]
I believe this concern is addressed by #562. Closing, but let's talk if I'm mistaken.

**Comment by afred** [comment]
:+1:
